### DISCUSSION TITLE
i3blocknextmatch : Next Match Schedule on i3blocks

### DISCUSSION THREAD

**Discussion by akshaypose** [i3/i3#5844] [Show and tell]
https://github.com/akshaypose/i3blocknextmatch
 An i3blocks blocklet which let's you see the next scheduled match details for your football team.
![image](https://github.com/i3/i3/assets/20870860/0b1cb498-0b37-4970-8161-b3ffc7b07ab5)

**Comment by mikeredev** [comment]
nice. this is a great way to get into working with i3blocks. did you know those blocks support pango markup too? so you can adjust the font, or add glyphs
